src/components/TopBar/TopBar.tsx

```diff
@@ -5,7 +5,7 @@ import useDeviceInfo from 'src/utils/useDeviceInfo';
 
 const useStyles = makeStyles((theme: Theme) => ({
   appBar: {
-    background: theme.palette.TwClrBg,
+    background: '#FBF9F9',
```

Comment:
Is this color going to be used many times? We could ask Chudo to create a token for this

Reply:
Good call, I will ask him when he's in today

Reply:
@charlie-tf this brings up a bigger question - should we have the token values in place for visual updates - in order to develop?
Would be great to keep the tokens as-is in terraware-web - so as not to undo your replace-hardcoded-values effort.
Should we work with @chudo76 to generate new zeroheight links for web-components 2.x.x?
As a hack, we could replace the values in the theme file - in web-components 2.x.x - until we have updated zeroheight files for web-components 2.x.x. That way, terraware-web can remain unaffected.
Thoughts?

Reply:
A separate theme for visual-updates in web-components 2.x.x sounds like a good way to go 👍

Reply:
Would it feel okay to merge this as is and circle back to update those hex values with tokens when they are in place?